Directory.Build.props

```diff
@@ -8,21 +9,9 @@
     <DebugType>embedded</DebugType>
     <DebugSymbols>true</DebugSymbols>
     <LangVersion>Latest</LangVersion>
-    <IsPackable>true</IsPackable>
     <EnablePackageValidation>true</EnablePackageValidation>
 
-    <CommonPath>$(MSBuildThisFileDirectory)src/Common/</CommonPath>
-  </PropertyGroup>
-
-  <PropertyGroup Condition="'$(IsTestProject)' == 'true'">
-    <IsPackable>false</IsPackable>
-    <TestCaptureOutput>false</TestCaptureOutput>
-    <TargetPlatform Condition="'$(TargetPlatform)' == ''">x64</TargetPlatform>
+    <CommonPath>$([MSBuild]::NormalizeDirectory('$(RepoRoot)', 'src', 'Common'))</CommonPath>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(IsTestProject)' == 'true'">
-    <PackageReference Update="Microsoft.NET.Test.Sdk" Version="$(MicrosoftNETTestSdkVersion)" PrivateAssets="true" />
-    <PackageReference Update="xunit" Version="$(XunitVersion)" />
-    <PackageReference Update="xunit.runner.visualstudio" Version="$(XUnitRunnerVisualStudioVersion)" PrivateAssets="all" />
```

Comment:
Why is this code not needed anymore? Is it all default in Microsoft.DotNet.Arcade.Sdk? Or different reason?

Reply:
Yes, this is already handled by Arcade in Tests.targets and xunit.targets.